stix_shifter_modules/trellix_endpoint_security_hx/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,77 @@
+{
+  "ipv4-addr": {
+    "fields": {
+      "value": ["Local IP Address", "Remote IP Address"]
+    }
+  },
+  "ipv6-addr": {
+    "fields": {
+      "value": ["Local IP Address", "Remote IP Address"]
+    }
+  },
+  "network-traffic": {
+    "fields": {
+      "src_port": ["Local Port"],
+      "dst_port": ["Remote Port"],
+      "src_ref.value": ["Local IP Address"],
+      "dst_ref.value": ["Remote IP Address"],
+      "extensions.'http-request-ext'.request_header.'Accept-Encoding'": ["HTTP Header"],
+      "extensions.'http-request-ext'.request_header.'User-Agent'": ["HTTP Header"],
+      "extensions.'http-request-ext'.request_header.Host": ["HTTP Header"],
+      "extensions.'http-request-ext'.request_value": ["URL"]
+    }
+  },
+  "user-account": {
+    "fields": {
+      "user_id": ["Username"]
+    }
+  },
+  "windows-registry-key": {
+    "fields": {
+      "key": ["Registry Key Full Path"],
+      "values[*].name": ["Registry Key Value Name"],
+      "values[*].data": ["Registry Key Value Text"]
+    }
+  },
+  "domain-name": {
+    "fields": {
+      "value": ["DNS Hostname"]
+    }
+  },
+    "file": {
+    "fields": {
+      "name": ["File Name"],
+      "size": ["Size in bytes"],
+      "hashes.MD5": ["File MD5 Hash"],
+      "parent_directory_ref.path": ["File Full Path"],
+      "x_path": ["File Full Path"]
+    }
+  },
+  "directory": {
+    "fields": {
+      "path": ["File Full Path"]
```

Comment:
if this is a directory not a file that should be updated to reflect that.

Reply:
Trellix doesnt have separate field for directory path of the file. The file full path which is mapped holds the value of  directory  of the file and file name